packit_service/worker/handlers/abstract.py

```diff
@@ -357,14 +345,30 @@ def _clean_workplace(self):
             else:
                 shutil.rmtree(item)
 
-    def pre_check(self) -> bool:
-        """
-        Implement this method for those handlers, where you want to check if the properties are
-        correct. If this method returns False during runtime, execution of service code is skipped.
+    @staticmethod
+    def get_checkers() -> Tuple:
```

Comment:
I think you should be able to go with `Sequence[Type[Checker]]` too… And since this comment is 8 days old, I'm not sure if it was the best idea to go commit-by-commit :D